orders/models.py

```diff
@@ -0,0 +1,86 @@
+from django.db import models
+from django.utils import timezone
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    order_id = models.AutoField(primary_key=True)
+    dog_owner_id = models.ForeignKey(DogOwner, on_delete=models.DO_NOTHING)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.DO_NOTHING)
+    booking_date = models.DateTimeField(auto_now=True)
+    approving_date = models.DateTimeField(blank=True, null=True)
+    starting_date = models.DateTimeField()
+    finishing_date = models.DateTimeField()
+    canceling_date = models.DateTimeField(blank=True, null=True)
+    total_price = models.IntegerField()
+    status = models.CharField(max_length=1,
+                              choices=StatusOptions.choices,
+                              default='P'
+                              )
+
+    def __str__(self):
+        return f"Dog owner ID: {self.dog_owner_id}, daycare ID: {self.daycare_id}, status: {self.status}"
+
+    @staticmethod
+    def create(dog_owner_id, daycare_id, starting_date, finishing_date, total_price):
```

Comment:
why do we need the create function?

Reply:
> why do we need the create function?

The dog owner chooses the dates of a requested order, and when he decides to place the order, the order needs to be created and then saved in the database with the fields mentioned.
Do you think we should try something different?